boards/arm/reel_board/reel_board-pinctrl.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	pwm0_default: pwm0_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, -1)>;
```

Comment:
Something went wrong here.

Reply:
fixed, added `nordic,invert` (board was only doing that).